tests/suites/test_suite_mpi.data

```diff
@@ -681,41 +681,23 @@ mpi_core_lt_ct:"11FFFFFFFFFFFFFFFF":"FF1111111111111111":1
 mbedtls_mpi_core_lt_ct: x>y (alternating limbs)
 mpi_core_lt_ct:"FF1111111111111111":"11FFFFFFFFFFFFFFFF":0
 
-mbedtls_mpi_core_cond_assign: same size, true condition
-mpi_core_cond_assign:"FFFFFFFF":"11111111":1
+mbedtls_mpi_core_cond_assign
+mpi_core_cond_assign:"FFFFFFFF":"11111111"
```

Comment:
I'd like to have a few more test cases to rule out some plausible implementation mistakes, like only copying the first limb, or copying the bytes in the wrong order, or doing bitwise operations wrong (e.g. `destination |= source` instead of `destination = source`). So at least one test case with ≥65 bits (in addition to at least one test case with the minimum number of limbs, which is 1), and test cases where the bytes aren't all the same and all four bit combinations at a given position {(0,0), (0,1), (1,0), (1,1)} are represented.

Goes for both assign and swap.